packages/cache/src/internal/cacheHttpClient.ts

```diff
@@ -122,6 +125,32 @@ export async function getCacheEntry(
   return cacheResult
 }
 
+async function listCache(
+  key: string,
+  httpClient: HttpClient,
+  version: string
+): Promise<void> {
+  const resource = `caches?key=${encodeURIComponent(key)}`
```

Comment:
does the actions run time token have permissions to call this API?

Reply:
From my testing no: [workflow run](https://github.com/Phantsure/test-repo/actions/runs/3444578591/jobs/5747330201). But is there a way to have those permissions? or is it not recommended from security perspective?

Reply:
We will have to give explicit permission to the token and I think it may not be desirable to do so. Unfortunately we only have a "manage" permission and not more granular like "read".